src/futhark.nim

```diff
@@ -667,10 +705,13 @@ macro importcImpl*(defs, outputPath: static[string], compilerArguments, files, i
     # This triggers an error in the example, it needs to be compiled twice for it to work
     #if node["kind"].str == "const":
     #  state.used.incl node["name"].str
-    var shouldImport = node["file"].str in extraFiles
+    var nodefile = node["file"].str
```

Comment:
I read this as no-defile, should probably be renamed to `nodeFile` for clarity..